PowerEditor/src/NppCommands.cpp

```diff
@@ -3594,6 +3594,12 @@ void Notepad_plus::command(int id)
 		}
 		break;
 
+		case IDC_REFRESH_WORK_DIR:
+		{
+			setWorkingDir(_pEditView->getCurrentBuffer());
+		}
+		break;
+

```

Comment:
Except some legacy code, `NppCommands.cpp` contains only the user commands (ie. menu commands).
The above code should be rather in `NppBigSwitch.cpp`

Reply:
I see, thanks.